lib/shared/bucketing-assembly-script/assembly/managers/configDataManager.ts

```diff
@@ -0,0 +1,15 @@
+import { ConfigBody } from '../types'
+
+let _configData: ConfigBody | null = null
+
+export function _setConfigData(configData: ConfigBody): void {
```

Comment:
How do the existing interfaces for the Server SDKs work? do they only support a single project? Or can you create multiple instances for different projects?

Reply:
the client itself needs to be instantiated with a single project token, so if you wanted to use multiple projects you'd have to make multiple clients. I guess this wouldn't support that since it's basically a global singleton

Reply:
so should the signature for the generateBucketedConfig require a token as well then?

Reply:
Yea it would need to.